TCGA Consensus and Comparison Revised (1 of 2) (#562)
AlexsLemonade/OpenPBTA-analysis: analyses/snv-callers/README.md

```
bash run_caller_analysis.sh
```

> For the TCGA data, it has its own script to run the same methods:

Reviewer: ```suggestion
The TCGA data is processed by its own script to run the same methods:
```

```
bash run_caller_analysis-tcga.sh
```

This bash script will return:

- Comparison plots in a notebook: [`compare_snv_callers_plots.nb.html`](https://cansavvy.github.io/openpbta-notebook-concept/snv-callers/compare_snv_callers_plots.nb.html).
- A zip file containing:

> - `[pbta|tcga]-snv-consensus-mutation.maf.tsv` - [MAF-like files](#consensus-mutation-call) that contain the snvs that were called by all callers described below for a given sample.

Reviewer: ```suggestion
- `(pbta|tcga)-snv-consensus-mutation.maf.tsv` - [MAF-like files](#consensus-mutation-call) that contain the snvs that were called by all callers described below for a given sample.
```

>   - `pbta/tcga-snv-consensus-mutation.maf.tsv` - is  [MAF-like file](#consensus-mutation-call) that contains the snvs that were called by all three of these callers for a given sample are saved to this file.

Reviewer: ```suggestion
  - `[pbta|tcga]-snv-consensus-mutation.maf.tsv` - [MAF-like files](#consensus-mutation-call) that contain the snvs that were called by all three callers (mutect2, strelka2, and lancet) for a given sample.
```

>   - `pbta/tcga-snv-consensus-mutation.maf.tsv` - is  [MAF-like file](#consensus-mutation-call) that contains the snvs that were called by all three of these callers for a given sample are saved to this file.

Reviewer: ```suggestion
  - `[pbta|tcga]-snv-consensus-mutation.maf.tsv` - [MAF-like files](#consensus-mutation-call) that contain the snvs that were called by all callers described below for a given sample.
```

### Mutation Comparisons

The consensus mutations called are those that are shared among all of Strelka2, Mutect2, and Lancet.
Mutations were considered to be the same if they were identical in the following field: `Chromosome`, `Start_Position`, `Reference_Allele`,  `Allele`, and `Tumor_Sample_Barcode`.
As Strelka2 does not call multinucleotide variants (MNV), but instead calls each component SNV as a separate mutation, MNV calls from Mutect2 and Lancet were separated into consecutive SNVs before comparison with Strelka2.

### Tumor Mutation Burden Calculation

For each experimental strategy and TMB calculation, the intersection of the genomic regions effectively being surveyed are used.

> Only Strelka2 and Mutect2 agreement was used for TMB calculations. This is because of some [complications identified with Lancet's performance on WXS data](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/snv-callers/lancet-wxs-tests).

Reviewer: ```suggestion
Only Strelka2 and Mutect2 agreement were used for TMB calculations. 
This is because of some [complications identified with Lancet's performance on WXS data](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/snv-callers/lancet-wxs-tests).
```

> These genomic regions are used for first filtering mutations to these regions and then for using the size in bp of the genomic regions surveyed as the TMB denominator.

Reviewer: ```suggestion
These genomic regions are used for first filtering mutations to these regions and then for calculating the size of the genomic regions surveyed as the TMB denominator.
```

> SNVs outside of these coding sequences are filtered out before being summed and used for TMB calculations like such:

Reviewer: ```suggestion
SNVs outside of these coding sequences are filtered out before being summed and used for TMB calculations as follows:
```
